charts/redpanda/values.go

```diff
@@ -345,18 +345,10 @@ type RedpandaResources struct {
 }
 
 func (rr *RedpandaResources) GetOverProvisionValue() bool {
-	// TODO restore this implementation after release.
-	// The RedpandaSMP function can be executed after this function, so that
-	// overprovisoned flag might not be set
-	//if rr.CPU.Overprovisioned == nil {
-	//	return false
-	//}
-	//
-	//if int(rr.RedpandaCoresInMillis()) < 1000 {
-	//	return true
-	//}
-	//
-	//return *rr.CPU.Overprovisioned
+	if int(rr.RedpandaCoresInMillis()) < 1000 {
```

Comment:
There's a really interesting issue from @StephanDollberg on what the correct way to set this is. Curious if he (currently) has any suggestions on what we should be doing here.

Reply:
I am not sure I understand the full PR but this change only seems to be concerned with setups where we have less than one 1 core/1000 millicpus assigned to Redpanda?

In that case using overprovisioned unconditionally is the right thing to do.

Reply:
That's exactly what it is, I just didn't understand at the time 😓 

Thank you for confirming! I was going to seek further guidance on the `--overprovisioned` flag but we'll do that at a later time.